R/simulate.r

```diff
@@ -124,6 +124,7 @@
 #' "Branching Process Models for Surveillance of Infectious Diseases
 #' Controlled by Mass Vaccination.” Biostatistics (Oxford, England)
 #' 4 (2): 279–95. \doi{https://doi.org/10.1093/biostatistics/4.2.279}.
+# nolint start: cyclocomp_linter
```

Comment:
```suggestion
# nolint start: cyclocomp_linter.
```

The regex for the specific exclusion of linters is a bit fussy and requires a period at the end. Otherwise, it's going to exclude all linters.

Reply:
Oh, thanks for catching that.